chia/wallet/wallet.py

```diff
@@ -1,6 +1,6 @@
 import logging
 import time
-from typing import Any, Dict, List, Optional, Set
+from typing import Any, List, Optional, Set, Dict
```

Comment:
isort is going to complain about this now.

```suggestion
from typing import Any, Dict, List, Optional, Set
```